src/services/SyncService.js

```diff
@@ -91,17 +91,22 @@ class SyncService {
 	}
 
 	async open({ fileId, initialSession }) {
-		this.on('change', ({ sessions }) => {
+		const onChange = ({ sessions }) => {
 			this.sessions = sessions
 		})
```

Comment:
```suggestion
		const onChange = ({ sessions }) => {
			this.sessions = sessions
		}
```

Reply:
Thanks! :heart: 
And that is why one should always test again after making changes - even if they seem simple :wink: